cookbooks/aws-parallelcluster-install/files/default/base/ami_cleanup.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
+# clean up cloud init artifacts https://cloudinit.readthedocs.io/en/latest/topics/cli.html#clean
+cloud-init clean -s
```

Comment:
The `-s` options removes an additional folder that wasn't being removed until now. is this desired?

Reply:
yes, there were also other folders that weren't removed before. The command (plus the flag) will remove all of them. This is the desired since we don't want any metadata of the building image to be stored into the AMI.